packages/forms-web-app/src/lib/views.js

```diff
@@ -44,7 +44,7 @@ const VIEW = {
 		AGENT: {
 			REPRESENTING_FOR: 'register/agent/who-representing',
 			REPRESENTEE_NAME: 'register/agent/name-person-representing',
-			REPRESENTEE_NAME_FAMILY: 'register/agent/name-family-group-representing',
+			REPRESENTEE_NAME_FAMILY: 'register/agent/name-household-representing',
```

Comment:
since we are changing the value, it would also make sense to change the variable name to better describe what it represents